src/linux/wpa-controller/ProtocolHostapd.cxx

```diff
@@ -85,3 +86,55 @@ Wpa::WpaPreSharedKeyPropertyKeyAndValue(const WpaPreSharedKey& wpaPreSharedKey)
     auto propertyValue = WpaPreSharedKeyPropertyValue(wpaPreSharedKey);
     return std::make_pair(propertyName, std::move(propertyValue));
 }
+
+std::vector<WpaKeyManagement>
+Wpa::WpaKeyManagementFromPropertyValue(std::string_view wpaKeyManagementProperty) noexcept
+{
+    std::string wpaKeyManagementString(wpaKeyManagementProperty);
+    std::istringstream wpaKeyManagementStream(wpaKeyManagementString);
+    std::vector<WpaKeyManagement> wpaKeyManagements{};
+
+    for (std::string wpaKeyManagement; wpaKeyManagementStream >> wpaKeyManagement;) {
```

Comment:
I should have mentioned this earlier, but I think there is value in separating the conversion of a single string to a single `WpaKeyManagement` value (ie. factor out the code within the for loop here into its own function, then just call that from here). This way we could write a test to validate the round-trip on the conversion is accurate, eg. 

```C++
for (auto wpaKeyManagement : AllWpaKeyManagements) {
    REQUIRE(WpaKeyManagementFromProperryValue(WpaKeyManagementPropertyValue(wpaKeyManagement)) == wpaKeyManagement);
}
```

This will ensure the corresponding conversion functions are at least consistent. Though, this is pretty trivial to change, so let's just keep what you have, and I can send a PR with that change tomorrow.